tests/backup_helper.go

```diff
@@ -8752,3 +8752,59 @@ func ValidatePVCCleanup(pvcBefore, pvcAfter []string) error {
 
 	return nil
 }
+
+type MaintenanceJobType string
+
+const (
+	FullMaintenanceJob  MaintenanceJobType = "full-maintenance-repo"
+	QuickMaintenanceJob MaintenanceJobType = "quick-maintenance-repo"
+)
+
+// ModifyMaintenanceJobFrequency modifies the frequency of the cron job
+func ModifyMaintenanceJobFrequency(backupLocationName string, frequencyInMinutes int, maintenanceJobType MaintenanceJobType) error {
+	k8sBatch := batch.Instance()
+	cronJob, err := GetCronJobByBackupLocation(backupLocationName, maintenanceJobType)
+	if err != nil {
+		return err
+	}
+	cronJob.Spec.Schedule = fmt.Sprintf("*/%d * * * *", frequencyInMinutes)
+	_, err = k8sBatch.UpdateCronJob(cronJob)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// GetCronJobByBackupLocation takes the backup location name and the job type and returns the cron job
+func GetCronJobByBackupLocation(backupLocationName string, maintenanceJobType MaintenanceJobType) (*batchv1.CronJob, error) {
```

Comment:
Can we take ctx as an argument?

Reply:
This is not user specific. So ctx does not make sense here. `ctx` is need to fetch all the available backup locations which can will be given by admin context.

Reply:
Makes sense. Thank you.